common/sketch/src/main/java/org/apache/spark/util/sketch/BloomFilterImpl.java

```diff
@@ -266,6 +266,12 @@ public static BloomFilterImpl readFrom(InputStream in) throws IOException {
     return filter;
   }
 
+  public static BloomFilterImpl readFrom(byte[] bytes) throws IOException {
+    try (ByteArrayInputStream bis = new ByteArrayInputStream(bytes)) {
```

Comment:
`ByteArrayInputStream` does not need to be closed, but the habit of using `try-with-resources` has been maintained here